osbs/conf.py

```diff
@@ -266,12 +267,22 @@ def get_registry_secrets(self):
         else:
             return []
 
-    def get_registry_api_versions(self):
+    def get_registry_api_versions(self, platform=None):
         value = self._get_value("registry_api_versions",
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        return value.split(',')
+        if not platform:
+            return value.split(',')
+        else:
+            self.get_platform_descriptors()
```

Comment:
How about just looking up the section directly?
```python
section = 'platform:{0}'.format(platform)
enable_v1 = self._get_value("enable_v1", section, "enable_v1", False, True)
```

`_get_value` should handle if either the config section or option is not defined.